plugins/kubernetes-backend/src/service/getKubernetesObjectsByServiceIdHandler.ts

```diff
@@ -34,7 +34,6 @@ const DEFAULT_OBJECTS = new Set<KubernetesObjectTypes>([
   'pods',
   'services',
   'configmaps',
-  'secrets',
```

Comment:
In order to not give people Authorization nightmares before we have that system in place. I think we should de-scope secrets until we have a separate strategy for viewing them safely.

Reply:
Yep, sounds good